### PR TITLE
fix: updated stack definition

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Updated stack definition to have input for `existing_kms_instance_crn`.
### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — CHANGES_REQUESTED** [review]
The new variable needs to be added into `ibm_catalog.json`

**Review by ocofaigh — APPROVED** [review]

File: ibm_catalog.json [hunk 1]
```diff
@@ -277,6 +277,13 @@
 							"default_value": "https://github.com/IBM/gen-ai-rag-watsonx-sample-application",
 							"description": "The URL to the public git repository containing the sample rag application code.",
 							"required": false
+						},
+						{
+							"key": "existing_kms_instance_crn",
+							"type": "string",
+							"default_value": "__NULL__",
+							"description": "The CRN of an existing KMS instance to use in this solution. If not set, a new KP instance is provisioned.",
```
Comment (vburckhardt) [inline ibm_catalog.json:285]: KP -> Key Protect

**Pull request merged**